ci/release/update-version.sh

```diff
@@ -56,3 +56,14 @@ sed_runner "/TAGFILES/ s|[0-9]\+.[0-9]\+|${NEXT_SHORT_TAG}|g" cpp/doxygen/Doxyfi
 for FILE in .github/workflows/*.yaml; do
   sed_runner "/shared-action-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
 done
+
+# Need to distutils-normalize the original version
+CURRRENT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${CURRENT_SHORT_TAG}'))")
+NEXT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${NEXT_SHORT_TAG}'))")
+
+# Dependency versions in dependencies.yaml
+sed_runner "s/==${CURRRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" dependencies.yaml
+# Dependency versions in setup.py
+sed_runner "s/==${CURRRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" python/cuspatial/setup.py
+# Dependency versions in pyproject.toml
+sed_runner "s/==${CURRRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" python/cuspatial/pyproject.toml
```

Comment:
```suggestion
sed_runner "s/==${CURRRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" dependencies.yaml
# Dependency versions in setup.py
sed_runner "s/==${CURRRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" python/cuspatial/setup.py
# Dependency versions in pyproject.toml
sed_runner "s/==${CURRRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" python/cuspatial/pyproject.toml
```

Can we update these search patterns to not depend on the `CURRRENT_SHORT_TAG_PEP440` variable?

Depending on `CURRRENT_SHORT_TAG_PEP440` will prevent the lines from ever being updated again if they somehow get out of sync with this script (which does occasionally happen from time to time).

Also, looks like `CURRENT` has too many `R`'s in that env variable. But that's moot if it's gonna be removed here.

Reply:
What's the suggested change here? We need to match the PEP440 version string, so not sure how else to match that.